### PR TITLE
Update grpc-js version to v1.2.0

### PR DESCRIPTION
**Pull request by carlo-808**
## Proposed Release Notes
* Update `@grpc/grpc-js` to version v1.2.0

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -139,13 +139,14 @@
     "versioned": "npm run prepare-test && time ./bin/run-versioned-tests.sh"
   },
   "bin": {
-    "newrelic-naming-rules": "./bin/test-naming-rules.js"
+    "newrelic-naming-rules": "bin/test-naming-rules.js"
```
Comment (michaelgoin) [inline package.json:142]: what's up with this change?

File: package.json [hunk 2]
```diff
@@ -139,13 +139,14 @@
...
   },
   "dependencies": {
-    "@grpc/grpc-js": "1.1.8",
+    "@grpc/grpc-js": "^1.2.0",
```
Comment (michaelgoin) [inline package.json:145]: Are you intentionally not pinning the version now? We havent' really discussed if we are ready to do that yet or not.

File: package.json [hunk 3]
```diff
@@ -139,13 +139,14 @@
...
     "@grpc/proto-loader": "^0.5.5",
     "@newrelic/aws-sdk": "^3.0.0",
     "@newrelic/koa": "^5.0.0",
+    "@newrelic/native-metrics": "^6.0.0",
```
Comment (michaelgoin) [inline package.json:149]: ???

Comment (carlo-808) [reply package.json:149]: ooof!  I'll fix that.

File: package.json (hunk 2 above)
Comment (carlo-808) [reply package.json:145]: Good catch.  I'll pin that.

**Review by michaelgoin — APPROVED** [review]

**Pull request merged**